dask_jobqueue/core.py

```diff
@@ -26,13 +28,12 @@ class JobQueueCluster(Cluster):
     ----------
     name : str
         Name of Dask workers.
-    threads : int
-        Number of threads per process.
+    cores : int
+        Total number of cores on a node / job
+    total_memory: str
+        Total amount of memory on a node / job
```

Comment:
> if there is a use case for memory_per_worker then we should keep it. Is there such a use case?

I can't find such a use case. However this makes me think we should put in the comment here that the total_memory will be splitted among worker processes. There won't be a process handling all this memory alone, I think it is better if it is clear.